youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -44,6 +46,8 @@ def check_version(self):
                 self.basename, self.basename, required_version)
             if self._downloader:
                 self._downloader.report_warning(warning)
```

Comment:
The warning should be printed in `check_version`,  otherwise it's printed when we check it in `YoutubeDL`.

In that case, it may make more sense to rename `check_outdated` to something like `is_updated` or `is_outdated` (you'll have to negate it at least in one place so it's up to you to choose the name).